src/client/pages/Offer/Introduction/DetailsModal/Details/DanishDetails.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import * as Yup from 'yup'
+import { FormikProps } from 'formik'
+import { InputGroup, inputTypes } from 'components/inputs'
+import { birthDateFormats } from 'l10n/birthDateAndSsnFormats'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+
+import { QuoteInput } from '../types'
+import {
+  DetailInput,
+  TextInput,
+  AreaInput,
+  BooleanInput,
+  HomeOwnershipTypeInput,
+} from './components/DetailInput'
+import { Content, ContentColumn } from './components/Details.styles'
+
+export const DanishValidationSchema = Yup.object().shape({
+  firstName: Yup.string().required(),
+  lastName: Yup.string().required(),
+  birthDate: Yup.string()
+    .matches(birthDateFormats.backEndDefault)
+    .required(),
+  data: Yup.object({
+    livingSpace: Yup.number()
+      .min(1)
+      .required(),
+    householdSize: Yup.number()
+      .min(1)
+      .required(),
+    isStudent: Yup.boolean(),
+    subType: Yup.string().required(),
+  }).required(),
+})
+
+export const DanishDetails: React.FC<{
+  formikProps: FormikProps<QuoteInput>
+}> = ({ formikProps }) => {
+  const { birthDate } = useCurrentLocale()
+  return (
+    <Content>
+      <ContentColumn>
+        <InputGroup>
+          <TextInput
+            name="firstName"
+            label="DETAILS_MODULE_TABLE_FIRSTNAME_CELL_LABEL"
+            formikProps={formikProps}
+          />
+          <TextInput
+            name="lastName"
+            label="DETAILS_MODULE_TABLE_LASTNAME_CELL_LABEL"
+            formikProps={formikProps}
+          />
+          <DetailInput
```

Comment:
🤔 Wouldn't be the case of having a specific input for it, like BirthDateInput (not sure about the name)? I'm asking because looking at DetailInput file I can see that there is a more generic field that handle things that are common: DetailInput and then we have specialised fields that uses DetailInput under the hood...